largest-series-product.json

```diff
@@ -0,0 +1,119 @@
+        "Those are implementation details.",
+        "Testing them constrains implementations,",
+        "and not all implementations use these operations.",
+        "e.g. The implementation which makes a single pass through the digits."
+    ],
+    "cases": [
+        {
+            "description": "can find the largest product of 2 with numbers in order",
+            "digits": "0123456789",
+            "span": 2,
+            "expected": 72
+        },
+        {
+            "description": "can find the largest product of 2",
+            "digits": "576802143",
+            "span": 2,
+            "expected": 48
+        },
+        {
+            "description": "finds the largest product if span equals length",
+            "digits": "29",
+            "span": 2,
+            "expected": 18
+        },
+        {
+            "description": "can find the largest product of 3 with numbers in order",
+            "digits": "0123456789",
+            "span": 3,
+            "expected": 504
+        },
+        {
+            "description": "can find the largest product of 3",
+            "digits": "1027839564",
+            "span": 3,
+            "expected": 270
+        },
+        {
+            "description": "can find the largest product of 5 with numbers in order",
+            "digits": "0123456789",
+            "span": 5,
+            "expected": 15120
+        },
+        {
+            "description": "can get the largest product of a big number",
+            "digits": "73167176531330624919225119674426574742355349194934",
+            "span": 6,
+            "expected": 23520
+        },
+        {
+            "description": "can get the largest product of a big number II",
+            "digits": "52677741234314237566414902593461595376319419139427",
+            "span": 6,
+            "expected": 28350
+        },
+        {
+            "description": "can get the largest product of a big number (Project Euler)",
+            "digits": "7316717653133062491922511967442657474235534919493496983520312774506326239578318016984801869478851843858615607891129494954595017379583319528532088055111254069874715852386305071569329096329522744304355766896648950445244523161731856403098711121722383113622298934233803081353362766142828064444866452387493035890729629049156044077239071381051585930796086670172427121883998797908792274921901699720888093776657273330010533678812202354218097512545405947522435258490771167055601360483958644670632441572215539753697817977846174064955149290862569321978468622482839722413756570560574902614079729686524145351004748216637048440319989000889524345065854122758866688116427171479924442928230863465674813919123162824586178664583591245665294765456828489128831426076900422421902267105562632111110937054421750694165896040807198403850962455444362981230987879927244284909188845801561660979191338754992005240636899125607176060588611646710940507754100225698315520005593572972571636269561882670428252483600823257530420752963450",
```

Comment:
slight concern here - the number is quite large and the answer exceeds a 32-bit int (but fits well within a 64-bit int). Go is the only track that has this test case